packages/manager/modules/carbon-calculator/src/dashboard/carbon-footprint/service.js

```diff
@@ -0,0 +1,10 @@
+import moment from 'moment';
+import { DATE_FORMAT_PATTERN } from './constants';
+
+export default class CarbonFootprintService {
+  static computePreviousMonth() {
+    return moment()
+      .subtract(1, 'months')
+      .format(DATE_FORMAT_PATTERN);
```

Comment:
```suggestion
    const date = new Date();
    date.setMonth(date.getMonth() - 1);
    const dateFormatter = new Intl.DateTimeFormat(userLanguage, {month: 'long', year: 'numeric'});
    return dateFormatter.format(date);
```

Reply:
Maybe you don't need moment for doing this. Using native javascript date and format functions. And you can use user language in 'fr-FR' format to localize the output

Reply:
I thought about using translation files first but it's never used in the app that's why I used moment.

Reply:
no need to use translation files just take user language data inside injected user object and use it instead of `userLanguage` variable I use inside DateTimeFormat